tests/backend/netypne_model_importer_test.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import os
+import sys
+import logging
+import json, urllib.request
+
+import netpyne
+
+from netpyne import specs, sim
+from netpyne_ui.netpyne_model_interpreter import NetPyNEModelInterpreter
+import neuron
+import subprocess
+
+from netpyne_ui.netpyne_geppetto import NETPYNE_WORKDIR_PATH
+from netpyne_ui.netpyne_geppetto import NetPyNEGeppetto
+
+netpyne.__gui__ = False
+sys.path.insert(0, NETPYNE_WORKDIR_PATH)
+
+class TestNetPyNEModelInterpreter(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        HERE = os.path.dirname(os.path.realpath(__file__))
+        ROOT = os.path.dirname(HERE)
+        cls.path = NETPYNE_WORKDIR_PATH
+        modelpath = os.path.join(NETPYNE_WORKDIR_PATH, 'mod')
+        subprocess.call(["rm", "-r", os.path.join(modelpath, "x86_64")])
+        owd = os.getcwd()
+        os.chdir(modelpath)
+        p = subprocess.check_output(["nrnivmodl"])
+        os.chdir(owd)
+        try:
+            neuron.load_mechanisms(modelpath)
+        except:
+            logging.error("Error loading mechanisms", exc_info=True)
+
+    def test_dict_import_1(self):
+        print("------------------------------------")
+        print("Dictionary transform importModel:")
+        print("------------------------------------")
+        
+        # response = urllib.request.urlopen("https://www.opensourcebrain.org/projects/netpyneshowcase/repository/revisions/test_py36/raw/NetPyNE/UI/HHCellNetwork.txt.json")        
+        netpyne_info = {} #json.loads(response.read())
+
+        # print(type(netpyne_info['net']['params']))
+
+        netpyne_info['compileMod'] = False
+        netpyne_info['loadMod'] = False
+        netpyne_info['modFolder'] = "mod"
+        netpyne_info['netParamsPath'] = sys.path[1] + '/models'
```

Comment:
even though we don't support Windows atm, better use `os.path.join` instead of '/' to construct file path

Reply:
Fixed and pushed